lib/views/drawer/main_drawer_view.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:game_lib_app/blocs_and_cubits/auth/auth_bloc.dart';
+import 'package:game_lib_app/blocs_and_cubits/drawer_cubit.dart';
+import 'package:game_lib_app/blocs_and_cubits/user_cubit.dart';
+import 'package:game_lib_app/models/user.dart';
+import 'package:get/get.dart';
+
+class MainDrawerView extends StatefulWidget {
+  const MainDrawerView({Key? key}) : super(key: key);
+
+  @override
+  State<MainDrawerView> createState() => _MainDrawerViewState();
+}
+
+class _MainDrawerViewState extends State<MainDrawerView> {
+  bool _switchValue = Get.locale == const Locale('pl', 'PL');
+  void changeLanguage(bool value) {
+    if (Get.locale == const Locale('pl', 'PL')) {
+      Get.updateLocale(const Locale('en', 'US'));
+    } else {
+      Get.updateLocale(const Locale('pl', 'PL'));
+    }
+    if (mounted) {
+      setState(() {
+        _switchValue = value;
+      });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Builder(
```

Comment:
I think that you don't have to use builder when you move your bloc providers above material app. Please check it :)